src/components/map/Map.tsx

```diff
@@ -48,7 +48,13 @@ export const Map = (props: MapProps) => {
   ) => {
     if (!props.vehicles) return;
     const [longitude, latitude] = feature.geometry.coordinates;
-    props.vehicles.fetchVehicles({longitude, latitude});
+    const zoom = feature.properties.zoomLevel;
+    const range = getVisibleRange(feature.properties.visibleBounds);
+    props.vehicles.fetchVehicles({
+      coordinates: {longitude, latitude},
+      zoom,
+      radius: range / 2,
```

Comment:
Is there any limitation of this radius somewhere/somehow? If a user zooms out so all of Norway is visible we probably don't want all scooters in Norway returned from the query.

Reply:
This is handled by `MIN_ZOOM_LEVEL` in `use-vehicles.ts`